helm/mds/values.yaml

```diff
@@ -71,7 +91,7 @@ postgresql:
   hostReader:
   port: 5432
   existingSecret: mds-secrets
-  password: # for setting secret internally - not used by https://github.com/helm/charts/tree/master/stable/postgresql
```

Comment:
Why remove this comment?

Reply:
@invertigo i stil see a ```postgresql:

Reply:
@invertigo postgresql.password with clarifying comment are re-added